src/mumble/CMakeLists.txt

```diff
@@ -1144,3 +1132,44 @@ endif()
 target_include_directories(${TARGET_NAME} INTERFACE
   $<BUILD_INTERFACE:${AUTOGEN_INCLUDE_DIR}>
 )
+
+if(packaging)
+	set(overlay ON)
+	set(plugins ON)
```

Comment:
I don't think this is appropriate here :thinking: 
These should be defaulted where they are declared and then not overwritten just because we're packaging. I realize that this is what we had in the old code, but I'm not sure this is something we should keep :thinking:

Reply:
I believe they are forced because the installer code doesn't take into account whether the DLLs exist.

Reply:
It does for the overlay (at least there is a parameter) and maybe it would be best if it also did that for the plugins as well :thinking:

Reply:
Agreed, let's do it in a separate pull request.